### PR TITLE
In-progress Kinto deployment tests

### PR DESCRIPTION
**Pull request by chartjes**
Hoping to use this as a template for further tests.

r? @karlht

### CODE REVIEW

File: kinto/deploy-test/.cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "test_deployment.py::DeploymentTest::testChangeOfHash": true, 
+  "test_deployment.py::DeploymentTest::testCleanUp": true, 
+  "test_deployment.py::DeploymentTest::testSanityChecks": true
+}
```
Comment (pdehaan) [inline kinto/deploy-test/.cache/v/cache/lastfailed]: Did you mean to check in this `/.cache/` folder, or should that be in .gitignore?

**Disclaimer:** I have no idea what I'm talking about.

Comment (karlht) [inline kinto/deploy-test/.cache/v/cache/lastfailed]: I supect that should be in `.gitignore`

Comment (chartjes) [inline kinto/deploy-test/.cache/v/cache/lastfailed]: Yes I am dumb and inadvertently added that in.

**Comment by rpappalax** [comment]
doing some housekeeping on services-test.
@karlht, @chartjes is this ready for merge?

**Comment by chartjes** [comment]
I believe it is

**Comment by karlht** [comment]
Yes!

**Comment by rpappalax** [comment]
:+1:

**Pull request merged**